cmd/vclusterctl/cmd/platform/add/cluster.go

```diff
@@ -70,7 +70,7 @@ vcluster platform add cluster my-cluster
 		},
 	}
 
-	c.Flags().StringVar(&cmd.Namespace, "namespace", "loft", "The namespace to generate the service account in. The namespace will be created if it does not exist")
+	c.Flags().StringVar(&cmd.Namespace, "namespace", "vcluster-platform", "The namespace to generate the service account in. The namespace will be created if it does not exist")
```

Comment:
This breaks a lot of things as the Cluster Object is not updated with the namespace, I think https://github.com/loft-sh/vcluster/pull/2147 does this better

Reply:
I forgot about this when making this PR. I'll remove it from my PR and see if @zerbitx can help fix this one since I've found it.